spec/directions/bake_toc_spec.rb

```diff
@@ -102,6 +107,11 @@
                 <span class="os-text">Key Terms</span>
               </h2>
             </div>
+            <div class = "os-eoc os-references-container" data-type="composite-page" id="composite-page-3">
+              <h2 data-type="document-title">
+                <span class="os-text">References</span>
+              </h2>
+            </div>
```

Comment:
I think id for Key Terms should be changed to `id="composite-page-3"` because this one is added for References in line `58`.

Reply:
It was just copy/paste and I missed that. Thank you Natalia! :relaxed: